test/test-functions

```diff
@@ -9,6 +9,7 @@ LOOKS_LIKE_DEBIAN=$(source $os_release && [[ "$ID" = "debian" || " $ID_LIKE " =
 LOOKS_LIKE_ARCH=$(source $os_release && [[ "$ID" = "arch" || " $ID_LIKE " = *" arch "* ]] && echo yes || :)
 LOOKS_LIKE_SUSE=$(source $os_release && [[ " $ID_LIKE " = *" suse "* ]] && echo yes || :)
 KERNEL_VER=${KERNEL_VER-$(uname -r)}
+ARCH=$(uname -m)
```

Comment:
I think the current script allow callers to define it - not sure if it's used or not - perhaps ```ARCH=${ARCH-$(uname -m)}``` ?

Reply:
good catch thanks! just repushed with that fixed